docs/Contribute/ContributeTranslate.md

```diff
@@ -1,10 +1,15 @@
-# Translate BTCPay Server
+# Translating BTCPay Server
 
-BTCPay Server uses a translation platform called Transifex to allow users to translate BTCPay Server into more languages. The community is currently working on translating the BTCPay Server invoice checkout page and the BTCPay Server official website. 
+## Why translations matter
```

Comment:
This seems redundant now as we explain this in the h1, right?

Reply:
So you wan't the `Why Translations matter` title deleted ?

Reply:
If so, i'd like to keep it, as this is an attempt to create the template:
* Why XYZ matters
* How to contribute
* Tips/Questions to the community

If we keep the main template titles consistent in each `Contribute` document, it might be easier for readers to get accustomed to how they are ordered.

Reply:
Yes. "Why it mattters" does not need to be a section, it can be a single sentance in the introductory. H1 followed by h2 with no text in between is weird.

Reply:
In that case, a solution might be to add a summary of the page between H1 and h2 ?

Or better yet, move the little summary that's in `To make a translation contribution, follow these steps:` under H1 and expand it to include all the h2 titles ?

Reply:
Sure, you decide on the structure actually, since you began drafting this anyway. Not a blocker just a suggestion on my end. :) LMK when it's ready for merge.

Reply:
Resolved y 03e8766